rgw/v2/tests/s3_swift/test_encrypted_bucket_chown.py

```diff
@@ -67,79 +68,149 @@ def test_exec(config, ssh_con):
-                    s3_object_path = os.path.join(TEST_DATA_PATH, s3_object_name)
-                    log.info("s3 object path: %s" % s3_object_path)
-
-                    if config.test_ops.get("upload_type") == "multipart":
-                        log.info("upload type: multipart")
-                        reusable.upload_mutipart_object(
-                            s3_object_name,
-                            bucket,
-                            TEST_DATA_PATH,
-                            config,
-                            user2,
+                    log.info("creating bucket with name: %s" % bucket_name)
+                    bucket = reusable.create_bucket(bucket_name, rgw_conn1, user1)
+                    if config.test_ops.get("enable_version", False):
+                        log.info("enable bucket version")
+                        reusable.enable_versioning(
+                            bucket, rgw_conn1, user1, write_bucket_io_info
                         )
-                    else:
-                        log.info("upload type: normal")
-                        reusable.upload_object(
-                            s3_object_name,
-                            bucket,
-                            TEST_DATA_PATH,
-                            config,
-                            user2,
+                    # enable per bucket encryption on the bucket
+                    log.info(
+                        f"Encryption type is per-bucket, enable it on bucket : {bucket_name}"
+                    )
+                    encryption_method = config.encryption_keys
+                    sse_s3.put_bucket_encryption(
+                        s3_client1, bucket_name, encryption_method
+                    )
+                    # get bucket encryption
+                    log.info(f"get bucket encryption for bucket : {bucket_name}")
+                    sse_s3.get_bucket_encryption(s3_client1, bucket_name)
+
+                    # change owner of the bucket to user2
+                    log.info(
+                        f"user info for user1 is {user1['user_id']} and user2 is {user2['user_id']}"
+                    )
+                    log.info(
+                        f"change owner of the bucket {bucket_name} to {user2['user_id']}"
+                    )
+                    utils.exec_shell_cmd(
+                        f"radosgw-admin bucket list --uid={user1['user_id']}"
+                    )
+                    new_uid = user2["user_id"]
+                    tenant = "default"
+                    bucket = bucket_name
+                    reusable.link_chown_nontenant_to_nontenant(new_uid, bucket)
+                    bucket = reusable.create_bucket(bucket_name, rgw_conn2, user2)
+                    # create objects
+                if config.test_ops["create_object"] is True:
+                    # uploading data
+                    log.info("s3 objects to create: %s" % config.objects_count)
```

Comment:
Please use format

Reply:
sorry can you elaborate here ?

Reply:
Can we change this as  "log.info(f"s3 objects to create: {config.objects_count}")" , try to avoid %s and use f

Reply:
Updated in all the places , thanks for noticing this!
Pls review